Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/features/user/user-account-edit/user-account-edit.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { UserAccount } from 'src/app/models/UserAccount';
+import { ToastrService } from 'src/app/services/toastr.service';
+import { firstValueFrom } from 'rxjs';
+import { UserAccountService } from 'src/app/services/UserAccountService';
+
+@Component({
+  selector: 'app-user-account-edit',
+  templateUrl: './user-account-edit.component.html',
+  styleUrls: ['./user-account-edit.component.scss']
+})
+export class UserAccountEditComponent implements OnInit {
+  userAccountData!: UserAccount;
+  constructor(private router: Router,
+    private route: ActivatedRoute,
+    private userAccountService: UserAccountService,
+    private toastr: ToastrService) { }
+
+  ngOnInit(): void {
+    const id = Number(this.route.snapshot.paramMap.get('id'));
+    this.loadSession(id);
+  }
+
+  async loadSession(userAccountId: number) {
```

Comment:
I think this method name should be loadUserAccount

Reply:
Resolved